ecs-init/config/common.go

```diff
@@ -104,6 +104,9 @@ const (
 	// defaultCredentialsFetcherSocketPath is set to /var/credentials-fetcher/socket/credentials_fetcher.sock
 	// in case path is not passed in the env variable
 	DefaultCredentialsFetcherSocketPath = "/var/credentials-fetcher/socket/credentials_fetcher.sock"
+
+    //SecurityOptsEnvVar is the environment variable that specifies a list of string values to customize labels for MLS systems, such as SELinux
```

Comment:
is this a comma-separated or space-separated list?  Does it matter?

Reply:
It's a comma-separated list, will update this and thanks for raising this miss